package.json

```diff
@@ -37,6 +37,7 @@
     "debug": "^4.1.1",
     "dotenv": "^8.0.0",
     "express": "^4.17.1",
+    "faker": "^4.1.0",
```

Comment:
In my opinion, I think faker works to generate fake data and might not be really useful in our project. Why don't you add it under the devDependencies? I think it would be better there.

Reply:
Thank you